tls/s2n_config.c

```diff
@@ -94,6 +95,12 @@ static int s2n_config_init(struct s2n_config *config)
     config->ct_type = S2N_CT_SUPPORT_NONE;
     config->mfl_code = S2N_TLS_MAX_FRAG_LEN_EXT_NONE;
     config->accept_mfl = 0;
+    config->session_state_lifetime_in_nanos = S2N_STATE_LIFETIME_IN_NANOS;
+    config->use_tickets = 1;
```

Comment:
why is this the default?

Reply:
S2N_STATE_LIFETIME_IN_NANOS is the default currently. Didn't want to make any changes to that. 
(#define S2N_STATE_LIFETIME_IN_NANOS     21600000000)

Will default use_tickets to 0.